packages/fiori/src/themes/UploadCollection.css

```diff
@@ -11,6 +11,10 @@
 	background-color: var(--sapGroup_ContentBackground);
 }
 
+.uc-no-files-illustration {
+	pointer-events: none;
```

Comment:
It is strange that the text inside the illustrated message cannot be selected.

Reply:
Removed `pointer-events: none`. As you proposed, this should be fixed in the list item by using delegatesFocus.